simapp/sim_test.go

```diff
@@ -358,12 +367,44 @@ func BenchmarkFullAppSimulation(b *testing.B) {
 	app := NewSimApp(logger, db, nil, true, 0)
 
 	// Run randomized simulation
-	// TODO parameterize numbers, save for a later PR
-	_, err := simulation.SimulateFromSeed(getSimulateFromSeedInput(b, os.Stdout, app))
-	if err != nil {
-		fmt.Println(err)
-		b.Fail()
+	// TODO: parameterize numbers, save for a later PR
```

Comment:
?

Reply:
not sure what was this about. I just added the `:` for the VSCode `TODO:` highlighter